cypress/e2e/claim-portal-failure.cy.ts

```diff
@@ -18,12 +18,12 @@ describe("Claims Portal Failures", () => {
     });
 
     it("should handle no connected signer", () => {
-      cy.get("#additionalDetails", { timeout: 15000 }).should("be.visible").invoke("click");
+      cy.get(".additional-details", { timeout: 15000 }).first().should("be.visible").invoke("click");
```

Comment:
What is the motive behind changing all the `id` selectors to `class`?

Reply:
I had to change them because all the transactions will be displayed on the page at the same time, so they are no longer unique identifiers.